internal/provider/resource_consul_cluster_test.go

```diff
@@ -194,8 +262,27 @@ func TestAccConsulCluster(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckConsulClusterExists(resourceName),
 					resource.TestCheckResourceAttr(resourceName, "size", "SMALL"),
+					resource.TestCheckTypeSetElemNestedAttrs(resourceName, "ip_allowlist.*", map[string]string{
+						"address":     "172.25.14.0/24",
+						"description": "this is an updated IPV4 address",
+					}),
 				),
 			},
+			// Tests update failure for IP Allowlist with too duplicate CIDRs
```

Comment:
I'm happy to move all these tests in their own `TestAccConsulCluster_IPAllowlist` function, but this is likely going to increase CI time as resources  need to be created for the tests to run. If we prefer clarity, that makes sense. I was reading this [contributing guide](https://github.com/hashicorp/terraform-provider-hcp/blob/main/contributing/writing-tests.md#acceptance-tests-often-cost-money-to-run) and it seemed to prefer minimal resource creation.